packages/AdobeClient/src/AdobeClientSideKit.js

```diff
@@ -39,7 +39,7 @@ import { AdobeHbkConstructor } from '../../../HeartbeatKit/dist/AdobeHBKit.esm.j
             appMeasurement,
             settings,
             timestampOption,
-            isInitialized = false,
+            isClientInitialized = false,
```

Comment:
I know i said `isClientInitialized/isServerInitialized`, but i think that `isAdobeClientKitInitialized/isAdobeServerKitInitialized` is even more clear for someone who is new to this repo